package.json

```diff
@@ -15,6 +15,7 @@
     "coverage:report:ci": "nyc report --reporter=lcovonly || true",
     "dev": "webpack serve",
     "dev:coverage": "npm start & cypress open --env COVERAGE=open",
+    "cypress:nocoverage": "npm start & cypress run --env COVERAGE=false",
```

Comment:
`npm start` isn't needed here.  It's called by the orb.

Reply:
Hmm, would it be better to leave the npm start here and remove the start command from the orb? In case this is used elsewhere?